packages/experiments/src/components/Button/Button.styles.ts

```diff
@@ -1,4 +1,4 @@
-import { IButtonComponent } from './Button.types';
+import { IButtonComponent, IButtonStylesReturnType, IButtonTokenReturnType } from './Button.types';
```

Comment:
just out of curiousity is `IButtonTokenReturnType` really needed? Below things are defined as `IButtonComponent['tokens']`, shouldn't that be enough to imply the return value?

Reply:
Also curious to know the answer to this!

Reply:
Yes, it is really needed because of the TS issue (see PR description.) In fact, other changes you see with commenting out styles code were exposed because of this change.

The TS issue has been open since July of 2014. :/

Reply:
I've [known about this issue for some time](https://github.com/OfficeDev/office-ui-fabric-react/blob/9e82e7d8e8aee4a543bbcc3c180dc126822388d2/packages/experiments/src/utilities/IComponent.ts#L5) in trying to get rid of the need for devs to explicitly supply the full function signature (all args and return type.) At least the functional declaration works for all function args.